rb_ws/src/buggy/scripts/auton/autonsystem.py

```diff
@@ -63,6 +63,7 @@ def __init__(self,
 
         self.path_planner = PathPlanner(global_trajectory)
         self.other_steering = 0
+        self.rtk_status = 0
```

Comment:
Since we no longer check RTK status in init_check, this variable, and the associated subscriber and update callback should be removed